quiz/question.py

```diff
@@ -0,0 +1,23 @@
+"""
+Represents question module
+"""
+
+
+class Question:
+
+    def __init__(self, question_id, question_text):
+        """
+        Initialize question object
+        """
+        self.question_id = question_id
+        self.question_text = question_text
+
+    def is_correct(self, keys, answer):
+        """
+        Check if answer is correct
+        """
+        correct_answer = next((k for k in keys if self.question_id == k.key_id), None)
```

Comment:
rather than checking answer by taking all keys, we can extract answers at the time of parsing `questions.txt` and `key.txt`. So Question will have an `answer` property and `is_correct` will simple compare user entered answer with it.